examples/large_models/tp_llama/model-config.yaml

```diff
@@ -0,0 +1,23 @@
+#frontend settings
+minWorkers: 1
+maxWorkers: 1
+maxBatchDelay: 200
+responseTimeout: 300
+parallelType: "tp"
+deviceType: "gpu"
+
+torchrun:
+    nproc-per-node: 8
+
+handler:
+    converted_ckpt_dir: "PATH/TO/converted_checkpoints"
+    tokenizer_path: "/PATH/TO/MODEL/CHECKPOINTS/tokenizer.model"
+    model_args_path: "PATH/TO/model_args.json"
```

Comment:
you can remove "PATH/TO". In handler, you can append them to model_dir.

Reply:
I believe as we are not bundling the model weights so wont be able to use model_dir